bokeh/command/subcommands/serve.py

```diff
@@ -588,8 +608,15 @@ def invoke(self, args):
         # even if Tornado is not installed
         from bokeh.server.server import Server
 
-        argvs = { f : args.args for f in args.files}
-        applications = build_single_handler_applications(args.files, argvs)
+        files = []
+        for f in args.files:
+            if args.glob and '*' in f:
```

Comment:
The second clause is unnecessary, because by definition `--glob` tells us that the following arguments are globs and, more importantly, `*` is not the only special symbol in globs (e.g. `?`, `[0-9]`).

Reply:
Thanks!